WordPress/src/main/java/org/wordpress/android/ui/accounts/SitesListAdapter.java

```diff
@@ -0,0 +1,315 @@
+package org.wordpress.android.ui.accounts;
```

Comment:
This appears to be almost a copy of [SitePickerAdapter](https://github.com/wordpress-mobile/WordPress-Android/blob/feature/login-epilogue-screen/WordPress/src/main/java/org/wordpress/android/ui/main/SitePickerAdapter.java). This means if a bug is found in one adapter, we'll have to remember to fix it in the other, too. Can we simply reuse the existing adapter?